vortex-dtype/src/field.rs

```diff
@@ -177,7 +177,7 @@ impl FieldPath {
     }
 
     /// Does the field referenced by the field path exist in the given dtype?
-    pub fn exists(&self, dtype: DType) -> bool {
+    pub fn exists_in(&self, dtype: DType) -> bool {
```

Comment:
I've found these too to be somewhat confusing, so also suggesting these small naming changes